source/features/conversation-authors.css

```diff
@@ -8,3 +8,7 @@
 		margin-right: 2px !important;
 	}
 }
+
```

Comment:
We need a description for these "strange" rules. Particularly the PR title and a link to the PR/issue itself would be good.